### PR TITLE
Development bootstrap script

### PR DESCRIPTION
**Pull request by IdoElk**
Local debugging setup (FE + SQLite)

Doesn't include a full-setup with debugging capabilities.

### CODE REVIEW

File: devops/dev_bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
```
Comment (yammesicka) [inline devops/dev_bootstrap.sh:3]: Maybe add `-ux`?

File: devops/dev_bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (yammesicka) [inline devops/dev_bootstrap.sh:1]: Awesome job. Looks good on my part. @gal432 ?

File: devops/dev_bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  echo "Config already exists"
+fi
+
+echo "Installing prod requirements"
+pip3 install --user -r "${MAIN_FOLDER}/requirements.txt"
+echo "Installing dev requirements"
+pip3 install --user -r "${MAIN_FOLDER}/dev_requirements.txt"
+
+echo "Creating local SQLite DB"
+python3 "${DB_BOOTSTRAP_FILE_PATH}"
```
Comment (yammesicka) [inline devops/dev_bootstrap.sh:29]: Many distros have no `python3` executable. Check if `python3` exists, if not, try `python` instead  (same for `pip`)

File: devops/dev_bootstrap.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  echo "Creating config from template"
+  cp "${CONFIG_EXAMPLE_FILE_PATH}" "${CONFIG_FILE_PATH}"
+  echo "Writing secret key to config"
+  echo "SECRET_KEY = \"$(python -c 'import os;print(os.urandom(32).hex())')\"" >>"${CONFIG_FILE_PATH}"
+else
+  echo "Config already exists"
+fi
+
+echo "Installing prod requirements"
+pip3 install --user -r "${MAIN_FOLDER}/requirements.txt"
```
Comment (gal432) [inline devops/dev_bootstrap.sh:24]: why not venv in this folder?

File: devops/dev_bootstrap.sh [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+echo "Installing dev requirements"
+pip3 install --user -r "${MAIN_FOLDER}/dev_requirements.txt"
+
+echo "Creating local SQLite DB"
+python3 "${DB_BOOTSTRAP_FILE_PATH}"
+echo "Moving DB to main directory"
+mv "./db.sqlite" "${MAIN_FOLDER}/db.sqlite"
```
Comment (gal432) [inline devops/dev_bootstrap.sh:31]: sorry, I don't understand. why the DB need to change place?

File: devops/dev_bootstrap.sh [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+set +eux
```
Comment (gal432) [inline devops/dev_bootstrap.sh:33]: btw why?

File: devops/dev_bootstrap.sh (hunk 4 above)
Comment (gal432) [reply devops/dev_bootstrap.sh]: IMO if we are going to install packages in his computer, at least let's do it in a separate interpreter

File: devops/dev_bootstrap.sh (hunk 3 above)
Comment (gal432) [reply devops/dev_bootstrap.sh]: I don't think that our code supports python2 so it's not relevant, but as I said in the previous comment, we should install venv for this project in the dev-bootstrap

Comment (IdoElk) [reply devops/dev_bootstrap.sh]: My understanding from Yam's comment is that there are distros which python 3's executable is `python`. This is not meant to support python 2

**Pull request merged**